pkg/assembler/backends/testing/backend.go

```diff
@@ -25,12 +25,330 @@ import (
 
 type DemoCredentials struct{}
 
-type demoClient struct{}
+type demoClient struct {
+	packages []*model.Package
+}
+
+// Define some demo packages to test the query without also ingesting into Neo4j
+var demo demoClient
 
 func GetBackend(args backends.BackendArgs) (backends.Backend, error) {
-	return &demoClient{}, nil
+	demo = demoClient{
+		packages: []*model.Package{
```

Comment:
hmm, would it be possible to make a helper function for this, to create the structure, may make it a bit more readable.

Reply:
Moved to a separate file with utilities. Should be more readable now. PTAL

Reply:
the refactor looks good. Easier to read and keep adding more packages as needed.

Reply:
Thanks, resolving this to get this merged